base/timing.jl

```diff
@@ -104,13 +104,43 @@ function padded_nonzero_print(value, str)
     end
 end
 
+"""
+    formatbytes(bytes; digits::Int = 3)
+
+Converts from an integer number of bytes to a pretty printed string in nearest significant size units.
+
+```jldoctest
+julia> formatbytes(123)
+"123 bytes"
+
+julia> formatbytes(1234)
+"1.205 KiB"
+
+julia> formatbytes(1234567)
+"1.177 MiB"
+
+julia> formatbytes(1234567891)
+"1.150 GiB"
 
-function format_bytes(bytes) # also used by InteractiveUtils
+julia> formatbytes(1234567891234)
+"1.123 TiB"
+
+julia> formatbytes(1234567891234567)
+"1.097 PiB"
+
+julia> formatbytes(1234567891234567891)
+"1096.517 PiB"
+```
+
+!!! compat "Julia 1.6"
+    This function was an internal function named `format_bytes` prior to Julia 1.6.
+"""
+function formatbytes(bytes; digits::Int = 3) # also used by InteractiveUtils
```

Comment:
`bytes::Real`?

Or does it only support integer values?  Seems like it should support other Real types too.